domains/oslc-domains/pom.xml

```diff
@@ -38,23 +60,35 @@
                 <!-- End of user code
         -->
         <!-- General dependencies -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
```

Comment:
the libraries shall NEVER specify a logging backend, only the end user app does this (simple or log4j etc.)